### PR TITLE
Remove Parental lock

### PR DESCRIPTION
**Pull request by Kronos2308**

### CODE REVIEW

File: ParentalLockRemover.te [hunk 1]
```diff
@@ -0,0 +1,53 @@
+printf("Remove Parental Lock Kronos2308 -.-")
```
Comment (suchmememanyskill) [inline ParentalLockRemover.te]: Please remove your name from the script, for uniformity across the repository. You may however, add a line with # in the front (python style comment) with your name, if you wish (even though most of the code here is taken from the syswipe script it seems)

**Comment by suchmememanyskill** [comment]
Has this also been checked for side-effects? Nuking the entire save could cause other stuff to go wrong as well. The save belongs to 010000000000002e, which is [pctl](https://switchbrew.org/wiki/Parental_Control_services)

**Comment by Kronos2308** [comment]
I no side effects at the moment that save just appear when the parental lock is active and is erased when is off 
 
I tested on my switch and NP

Me an others use hack disk mount for get the same effect 


Use this has reference 
https://switchbrew.org/wiki/Flash_Filesystem#System_Savegames

**Comment by Kronos2308** [comment]
I think it's the best way is like this

but I can change it so it doesn't look like "system wipe"
if you wish

**Pull request merged**